safeeyes/glade/settings_dialog.glade

```diff
@@ -627,15 +567,53 @@
                                     <property name="position">1</property>
                                   </packing>
                                 </child>
+                                <child>
+                                  <object class="GtkBox" id="box16_random_order">
```

Comment:
Please move the random break option before the strict break to keep strict break and postpone breaks together (just to keep similar functionalities/options together for better UX)

Reply:
Will do